man/man8/radcrypt.8

```diff
@@ -9,9 +9,12 @@ radcrypt - generate password hash for use with radius, or validates a password h
 \fIplaintext_password\fP [\fIhashed_password\fP]
 .SH DESCRIPTION
 \fBradcrypt\fP generates a hashed digest of a plaintext password, or can
-validate if a password hash matches a plaintext password. DES and MD5
-hashes are currently supported. When generating a password hash a random
-salt is generated and applied.
+validate if a password hash matches a plaintext password. DES and MD5 hashes 
+are currently supported. bcrypt, SHA3, SHA2 and SSHA2, not to mention SSHA1 
+are not supported by radcrypt though they are supported by PAP. In other words, 
+radcrypt is essentially not maintained. When generating a password hash a random 
+salt is generated and applied. Custom salts are not supported (users should not
+be entering their own salts anyway).
```

Comment:
_I would suggest something along the lines of:_

radcrypt generates a hashed digest of a plaintext password, or can validate if a password hash matches a plaintext password. radcrypt only currently supports the older DES and MD5 hashes. Although more modern hashes such as bcrypt, SHA3, SHA2 and SSHA2, SSHA1 can not be generated by radcrypt, they can be generated by other tools and are supported in the server by rlm_pap(5).

When generating a password hash a random salt is generated and applied. Custom salts are not supported.